crate_universe/src/cli.rs

```diff
@@ -19,7 +19,11 @@ pub use splice::splice;
 pub use vendor::vendor;
 
 #[derive(Parser, Debug)]
-#[clap(name = "cargo-bazel", about, version)]
+#[clap(
+    name = "cargo-bazel",
+    about = "crate_universe` is a collection of tools which use Cargo to generate build targets for Bazel.",
+    version
+)]
```

Comment:
Note: this change, the the other changes to `#[clap(author ...` are not relevant to the underlying fix, but the changes were necessary to get the rust_doc_test test passing on Windows